shared/oae/api/oae.api.js

```diff
@@ -191,6 +186,12 @@ define(['underscore', 'oae.api.admin', 'oae.api.authentication', 'oae.api.config
                 // Perform any email verification if instructed before we try and determine if the
                 // user's profile info is valid
                 verifyEmail(function() {
```

Comment:
Can we swap `acceptInvitation` and `verifyEmail` in the order? In that case, we wouldn't have to do 

```
// We cannot accept an invitation as the anonymous user. Let the
// signup page handle it
if (oae.data.me.anon) {
  return callback();
}
```

in `acceptInvitation`